src/main/java/com/openshift/internal/restclient/ApiTypeMapper.java

```diff
@@ -141,12 +142,12 @@ private void addEndpoints(final String prefix, final String apiGroupName, final
 			}
```

Comment:
what if the name doesnt contain FWD_SLASH? we wont create a capability? is this right?